src/integrationTest/java/uk/gov/hmcts/reform/orgrolemapping/config/SwaggerPublisher.java

```diff
@@ -20,26 +20,31 @@
  * Built-in feature which saves service's swagger specs in temporary directory.
  * Each travis run on master should automatically save and upload (if updated) documentation.
  */
-@SpringJUnitWebConfig
-@SpringBootTest
-@AutoConfigureMockMvc
-class SwaggerPublisher {
+public class SwaggerPublisher extends BaseTest {
+
+    private MockMvc mockMvc;
 
-    @Autowired
-    private transient MockMvc mvc;
+    @Inject
+    private WebApplicationContext wac;
+
+    @Before
+    public void setUp() {
+        mockMvc = MockMvcBuilders.webAppContextSetup(wac).build();
+    }
 
     @DisplayName("Generate swagger documentation")
     @Test
     @Disabled
-    void generateDocs() throws Exception {
-        byte[] specs = mvc.perform(get("/v2/api-docs"))
-            .andExpect(status().isOk())
-            .andReturn()
-            .getResponse()
-            .getContentAsByteArray();
+    public void generateDocs() throws Exception {
+        byte[] specs = mockMvc.perform(get("/v3/api-docs"))
```

Comment:
Can we use constant for URL: `org.springdoc.core.Constants.DEFAULT_API_DOCS_URL`

Reply:
Resolved in 2451497b83d9cf77ae0c2e720efec1c96e3b4397